src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {

```

Comment:
u can read about Lombok library, it allows to avoid boilerplate code such as getters, setters, constructors, etc.

Reply:
ok, I will